native/packages/react-native-bpk-component-carousel/src/BpkCarouselItem.js

```diff
@@ -0,0 +1,43 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+
+import React, { type Node } from 'react';
+import PropTypes from 'prop-types';
+import { View, type StyleObj, ViewPropTypes } from 'react-native';
+
+export type Props = {
+  children: Node,
+  style: ?StyleObj,
+};
+
+const BpkCarouselItem = ({ children, style }: Props) => (
+  <View style={style}>{children}</View>
```

Comment:
Wouldn't it be good if these `View`s had `width: '100%'` and `height: '100%` as that's always what we want?

Reply:
Yeah I am wondering what the purpose of a `BpkCarouselItem ` is ...

Reply:
Tbf I advised having an item component, maybe in hindsight that wasn't the correct choice. I was thinking that it gives us a wrapper in case there's functionality we want to introduce after the fact.

Reply:
I don't know if `width/height: 100%` work with the scroll view. I can try again but as far as I remember it doesn't work.

Reply:
The `BpkCarouselItem` really seems to be without purpose here. I could replace it for a view but as @shaundon said it makes it much easier to evolve it in the future. e.g. 

```
<BpkCarouselItem onVisible={changeTab} />
```